sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_recognize_receipts.py

```diff
@@ -48,7 +48,7 @@ def recognize_receipts(self):
         receipts = poller.result()
 
         for idx, receipt in enumerate(receipts):
-            print("--------Recognizing receipt #{}--------".format(idx))
+            print("--------Recognizing receipt #{}--------".format(idx+1))
```

Comment:
can't comment on line 63, but do we want Items 1-indexed as well? if so, the readme should also be updated

Reply:
good catch, updated